### DISCUSSION TITLE
Thank you for this stellar lib

### DISCUSSION THREAD

**Discussion by zehawki** [table-library/react-table-library#47] [General]
What a great find! Thank you @rwieruch for investing your time and effort on this, and I totally get why you decided to create a new library. It exactly balances providing enough scaffolding OOB to get things up and running very quickly and then allowing serious control / customization via composability. I found everything to be of very high quality.

Thank you also for the wealth of documentation - storybook, tutorials etc. These were immensely helpful.

I've not been happy with MUI Data Grid for a variety of reasons, and have been wanting to replace it, and now I've done it across the entire product with react-table-library. It took me just 2 days to replace Data Grid with a more powerful table solution with a custom theme, resize, filter, search, sort, expand, multiple fields, multiple actions & multiple editing controls, csv download and local + server pagination. And I also ported over the column schema I'd created earlier with Data Grid, which allows me to control the table component without any hardcoding - ie whether the column is searchable, sortable & what method, showable, exportable, width, and what to render in the cell, etc, etc.

**Comment by rwieruch** [comment]
This is super kind of you @zehawki and it made my day. I do my best providing a greta feature parity in comparison to other table libraries while still keeping up this library's customization aspects :) If you have any feedback, hit me up!
